Microsoft.Toolkit.Uwp.SampleApp/readme.md

```diff
@@ -179,14 +179,22 @@ If the specified type is not found on the system running the sample app the samp
 
 ### Adding documentation
 
-Every API must be accompanied by Markdown documentation in the docs folder in the root of this repo.
+Every API must be accompanied by Markdown documentation in the [documentation repository](..\contributing.md#docs).
 
 Use the DocumentationUrl property to add a link to the raw documentation in *samples.json*. Please follow the following pattern:
 
-`https://raw.githubusercontent.com/Microsoft/WindowsCommunityToolkit/{branch}/docs/{folder/file.md}`
+`https://raw.githubusercontent.com/MicrosoftDocs/WindowsCommunityToolkitDocs/{branch}/docs/{folder/file.md}`
 
 > NOTE: When building and running the app in release mode, the branch will automatically be changed to **master** before loading. 
-> Note: The documentation is also packaged with the sample app. If there is no network connection, or the documentation is not yet on github, the sample app will use the packaged version
+
+> NOTE: The documentation is also packaged with the sample app. If there is no network connection, or the documentation is not yet on github, the sample app will use the packaged version
+
+> NOTE: To test your documentation in the sample app while running in debug mode, the docs repository will need to be cloned in the same folder as this repository and named **WindowsCommunityToolkitDocs**. For exaple, this folder structure works best:
```

Comment:
Why not add a submodule instead of requiring users to clone a second repo?

Reply:
Thought about that, and I'm still not against it if it makes sense. A submodule doesn't automatically get pulled in when doing only `git clone` and it's not the most discoverable command to do a recursive clone so it will still require additional work. It also seems difficult to keep the head of the submodule updated when trying to merge PRs across repos at the same time (I could be wrong here as I don't have experience with this)